core/iwasm/aot/aot_loader.c

```diff
@@ -2062,13 +2138,7 @@ load_relocation_section(const uint8 *buf, const uint8 *buf_end,
         }
 
         name_addr = symbol_buf + symbol_offsets[name_index];
-        str_len = *(uint16 *)name_addr;
-
-        if (!(group->section_name = const_str_set_insert(
-                  name_addr + sizeof(uint16), (int32)str_len, module, error_buf,
-                  error_buf_size))) {
-            goto fail;
-        }
+        read_string(name_addr, buf_end, group->section_name);
```

Comment:
doesn't the BH_PLATFORM_WINDOWS block above need similar changes to use read_string?

Reply:
Should no need to change it, the block directly uses the raw string to compare with the string pointer and length, without caring whether it's terminated with '\0' or not, without using strlen.